applications/asset_tracker/README.rst

```diff
@@ -77,28 +85,41 @@ LED 3 and LED 4:
 
 All LEDs (1-4):
     * Blinking in groups of two (LED 1 and 3, LED 2 and 4): Recoverable error in the BSD library.
-    * Blinking in cross pattern (LED 1 and 4, LED 2 and 3): Communication error with the nRF Cloud.
+    * Blinking in cross pattern (LED 1 and 4, LED 2 and 3): Communication error with nRF Cloud.
+
+On the Thingy:91, the application state is indicated by a single RGB LED as follows:
+
+.. csv-table:: Thingy:91 LED states
+   :header: "Color", "State"
+   "White", "Connecting to network"
+   "Cyan", "Connecting to nRF Cloud"
+   "Yellow", "Waiting for user association"
+   "Blue", "Connected, sending environment data"
+   "Purple", "Searching for GPS"
+   "Green", "Connected, sending GPS and environment data"
+   "Red", "Error"
 
 .. _power_opt:
 
 Power optimization
 ******************
 
-The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application.
+The Asset Tracker can run in three power modes that are configured in the Kconfig file of the application. These settings are currently only supported on the nRF9160 DK.
```

Comment:
To be a bit picky, Thingy:91 also runs  in PSM when the GPS has started

Reply:
It does. However, I don't think that detail belongs in the power optimization section. The device isn't really saving power in that case, as it is using the radio for GPS.